docs/tutorials/contracts/nfts/blockcraft-nfts.md

```diff
@@ -0,0 +1,171 @@
+
+To complete this tutorial successfully, you'll need:
+
+- A Minecraft account
+- WorldEdit [installed](https://worldedit.enginehub.org/en/latest/install/)
+- [A NEAR account](#wallet)
+- [NEAR command-line interface](/docs/develop/contracts/rust/intro#installing-the-near-cli) (`near-cli`)
+
+### Creating a Schematic
+
+Once you have WorldEdit installed and Minecraft loaded up, check if WorldEdit is working properly by running:
+
+```bash
+//pos1
+```
+
+If WorldEdit is properly installed, it should output `First position set to (X, Y, Z).` where X, Y, and Z are coordinates.
+
+For this tutorial, we will be minting a small village house. To follow along, choose any structure that you'd like to mint as shown below:
+
+> ![Village House Minecraft](/docs/assets/nfts/village-house-minecraft.png)
+
+You'll then want to choose the boundaries of the structure that you'd like to copy which we will turn into schematics. These schematics will be placed on chain for you or others to potentially download and paste in their own worlds. To do this, we'll need to outline the boundaries of the build using WorldEdit. Stand in the bottom left corner of your build and run:
+
+```bash
+//pos1
+```
+
+You can then move to the top right corner and run:
+
+```bash
+//pos2
+```
+
+Behind the scenes, this has created a cube around your build. We can then copy what we just selected using `//copy`. The output should look something like this:
+
+> ![Copy Chat Message](/docs/assets/nfts/copy-chat-message-minecraft.png)
+
+> **Tip:** Your player position matters when copying. If you copy the build and you're standing on the roof, when you paste the build, it will paste in a way that will result in you standing on the roof.
+
+## Sanity Check
+
+We can check and see if our build is fine by pasting what we just copied elsewhere in our world. Go to a location that you would like to paste the build and run `//paste`. In our case, we pasted the village house floating above a coral reef biome:
+
+> ![Pasted Minecraft House](/docs/assets/nfts/pasted-minecraft-house.png)
+
+### Minting your NFTs
+
+NEAR has already deployed a contract to the account `example-nft.testnet` which allows users to freely mint tokens. This is the account we'll be interacting with to mint our NFTs.
+
+Now let's mint our first token! The following command will mint one copy of your NFT.
+Please remember to replace the `token_id` value with an unique string.
+
+> **Tip:** you can also replace the `media` URL with a link to any image file hosted on your web server.
+
+```bash
+near call example-nft.testnet nft_mint '{"token_id": "my-token-unique-id", "receiver_id": "'$ID'", "token_metadata": { "title": "Some Art", "description": "My NFT media", "media": "https://upload.wikimedia.org/wikipedia/commons/thumb/0/00/Olympus_Mons_alt.jpg/1024px-Olympus_Mons_alt.jpg", "copies": 1}}' --accountId $ID --deposit 0.1
```

Comment:
Shouldn't we show how to upload the Minecraft structure somewhere, and use that URL for the NFT?